src/targets/python/requests.js

```diff
@@ -119,7 +119,12 @@ module.exports = function (source, options) {
 
   // Construct request
   const method = source.method;
-  let request = format('response = requests.request("%s", url', method);
+  let request;
+  if (['HEAD', 'GET', 'POST', 'PUT', 'PATCH', 'DELETE'].includes(method)) {
```

Comment:
method list pulled from their api reference https://docs.python-requests.org/en/latest/api/#requests.head

Reply:
Adding this as a code comment for future us.

```suggestion
  // Method list pulled from their api reference https://docs.python-requests.org/en/latest/api/#requests.head
  if (['HEAD', 'GET', 'POST', 'PUT', 'PATCH', 'DELETE'].includes(method)) {
```